library/core/src/fmt/rt.rs

```diff
@@ -23,38 +19,17 @@ pub struct Placeholder {
 #[cfg(bootstrap)]
 impl Placeholder {
     #[inline]
-    pub const fn new(
-        position: usize,
-        fill: char,
-        align: Alignment,
-        flags: u32,
-        precision: Count,
-        width: Count,
-    ) -> Self {
-        Self { position, fill, align, flags, precision, width }
+    pub const fn new(position: usize, flags: u32, precision: Count, width: Count) -> Self {
+        Self { position, flags, precision, width }
```

Comment:
This change was kind of awkward because #139131 changed the cfg's in the window between this PR and the beta branch. So the new bootstrap compiler is what `cfg(not(bootstrap))`  was pre-that-PR. cc @m-ou-se since you'd know better than me whether this seems right

Reply:
Yeah, this looks right.